CHANGELOG/1.2-CHANGELOG.md

```diff
@@ -0,0 +1,424 @@
+incompatible changes prior to the final release. The following represents the
+changes since v1.1.0:
+
+## Release Cycle changes
+
+In this release, we moved to using a phased release cycle modeled on the
+[upstream Kubernetes release cycle](https://kubernetes.io/releases/release/).
+
+There are now four phases:
+
+* **Scoping**: 4-6 weeks, where the maintainers and community determine the
+  set of features we want to include in the release. A key deliverable here
+  is ensuring that the Experimental channel does not get any bigger than it
+  already is; we want to ensure that features make their way to Standard or
+  are removed.
+* **GEP Iteration and Review**: 5-7 weeks, where we work to update GEPs, and
+  solidify and meet graduation criteria for in-scope GEPs.
+* **API Refinement and Documentation**: 3-5 weeks, where we translate proposed
+  changes from GEPs into changes to both the API specification and
+  documentation.
+* **SIG-Network Review and Release Candidates**: 2-4 weeks, where we perform
+  our required upstream reviews, get any further required changes in, and
+  release our release candidates so that implementations can get started
+  with work on the new features asap.
+
+For all the detail about this, please see the [Release Cycle docs](https://gateway-api.sigs.k8s.io/contributing/release-cycle/).
+
+Relevant PRs:
+* Adding Release Cycle Documentation by @robscott in [#3063](https://github.com/kubernetes-sigs/gateway-api/pull/3063)
+
+## Standard Channel Additions and Changes
+
+The Standard channel is Gateway API's set of maximally-stable install files.
+Only features with the best testing and support are added to the standard
+channel. This channel should be considered GA or stable, and future changes
+will only be compatible changes.
+
+### Infrastructure Labels and Annotations moved to Standard 🎉
+
+[GEP-1867](https://gateway-api.sigs.k8s.io/geps/gep-1867/) added an
+`infrastructure` stanza to Gateway objects that is intended to carry
+infrastructure configuration specific to that Gateway object.
+
+[GEP-1762](https://gateway-api.sigs.k8s.io/geps/gep-1762/) adds a section
+for `labels` and `annotations` to this stanza that specifies labels and
+annotations to be annotated to all Kubernetes resources created to fulfill
+that Gateway request.
+
+This feature can be used to affect the labels and annotations created on
+Loadbalancer Services by in-cluster implementations to fulfill the Gateway
+contract.
+
+This feature has graduated to Standard and is now considered GA or Stable.
+
+This feature's name for conformance tests and GatewayClass status reporting
+is `GatewayInfrastructurePropagation`.
+
+This feature's status is **Extended**, meaning that it is _optional_ for
+implementations to support. Please see the implementation's
+`gatewayclass.status.supportedFeatures` stanza or the 
+[implementation feature list](https://gateway-api.sigs.k8s.io/implementations/v1.2/)
```

Comment:
Recommend leaving this off since it usually takes us at least one month to populate the page.